### PR TITLE
Allow stickers to be attached to storages that aren't full

### LINKED ISSUE #16835: Unable to attach artifact analysis form to bag of holding artifact while active
### Labels

[BUG][MINOR]

### Description

title

### Steps to reproduce

activate a bag of holding
cannot attach form 
:(

### Expected Behavior

Should be able to stick to it 

### Additional Information & Screenshots

probably a bug focused around items with inventories considering it wont attach to a backpack either

Reported by: Gabr
On server: Goonstation 4 Roleplay: Sylvester
Active test merges: #16744 and #16370
Round log date: 2023-11-08-05-01
Reported on: 2023-11-08 05:14:52
Map: NADIR

### PR DESCRIPTION
**Pull request by FlameArrow57**
[GAME OBJECTS][PLAYER ACTIONS][QoL]

## About the PR 
This PR allows stickers to be attached to storages that aren't full by click dragging them onto the storage item

## Why's this needed? 
Currently you can only attach a sticker to a storage if its contents are full.

Fixes #16835

## Changelog 

```changelog
(u)FlameArrow57
(+)Stickers, including artifact forms, will now attach to storage items, rather than being added to their contents, when click dragging them onto the storage.
```

### CODE REVIEW

**Comment by Tarmunora** [comment]
intent-based non-combat interactions 😔

**Comment by FlameArrow57** [comment]
Alternate idea, stickers will always stick to backpacks if clicking them, and to add them to a backpack, you click the sticker with the backpack?

**Comment by Tarmunora** [comment]
🤷

**Comment by pali6** [comment]
> Alternate idea, stickers will always stick to backpacks if clicking them, and to add them to a backpack, you click the sticker with the backpack?

You can still click an empty space in the backpack UI with the sticker, right? That plus clicking the sticker with a backpack feels like enough ways to get the sticker in. I'd get rid of the intent thing.

**Comment by FlameArrow57** [comment]
> > Alternate idea, stickers will always stick to backpacks if clicking them, and to add them to a backpack, you click the sticker with the backpack?
> 
> You can still click an empty space in the backpack UI with the sticker, right? That plus clicking the sticker with a backpack feels like enough ways to get the sticker in. I'd get rid of the intent thing.

To add it to the backpack, you click the UI. To attach it to the backpack, the only way is if the backpack is full. The problem is for artifact forms for bags of holding, which can have 20+ items, in which case it becomes really inconvenient

**Comment by FlameArrow57** [comment]
> > Alternate idea, stickers will always stick to backpacks if clicking them, and to add them to a backpack, you click the sticker with the backpack?
> 
> You can still click an empty space in the backpack UI with the sticker, right? That plus clicking the sticker with a backpack feels like enough ways to get the sticker in. I'd get rid of the intent thing.

Clicking an empty space counts as the backpack still being attacked by the sticker. Not sure about the solution now :/

**Comment by pali6** [comment]
Ah damn. I hoped the empty space click would bypass that.

File: code/obj/item/stickers.dm [hunk 1]
```diff
@@ -107,6 +108,12 @@
 			attached.visible_message("<span class='alert'><b>[src]</b> is destroyed!</span>")
 		..()
 
+	mouse_drop(atom/over_location)
```
Comment (Tarmunora) [inline code/obj/item/stickers.dm:111]: should probably rename to `over_object`, to match parent argument names (which also has an `over_location`, but as the *third* argument)

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/obj/item/stickers.dm (hunk 1 above)
Comment (FlameArrow57) [reply code/obj/item/stickers.dm]: Oh, oop

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**